test-specs/db/user-test.js

```diff
@@ -57,21 +74,24 @@ describe('User', () => {
 
       Promise.all([creatingUser, creatingAddress])
       .spread((user, address) => {
-        user.addShippingAddress([address])
-        user.addBillingAddress([address])
+        user.setShippingAddresses([address])
```

Comment:
Can a user store multiple shipping and/or billing addresses? If so, may be helpful to test that functionality